apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -651,25 +650,36 @@ export const getStudentsMCSummaryForCurrentAssessment = state => {
     const studentsAssessment =
       studentsObject.responses_by_assessment[currentAssessmentId];
 
-    // If the student has not submitted this assessment, display empty results.
+    /* In order for the sorting of the submission timestamp column to work correctly in the SubmissionStatusAssessmentsTable, the submissionTimeStamp field must be a Date. So, we pass in arbitrary Dates in the past to handle when the assessment is in progress or not yet started. */
```

Comment:
This is where things get a little weird. This seemed like a better workaround than writing a custom sort function, but let me know if you think it's too hacky or you have alternative ideas.